view/frontend/templates/iframe-handler.phtml

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+use CustomGento\Cookiebot\ViewModel\Script;
+use Magento\Framework\View\Helper\SecureHtmlRenderer;
+
+/** @var SecureHtmlRenderer $secureRenderer */
+/** @var Script $viewModel */
+$viewModel = $block->getData('view_model');
+
+// Prepare translations
+$acceptText = __('accept %1 cookies', 'marketing');
+$consentMessage = __('Please %1 to view this %2 content.');
```

Comment:
This needs to be one translation, not two. In other languages, the text might be completely different.